core-google-topic.atom.php

```diff
@@ -0,0 +1,14 @@
+<?php
+require_once __DIR__ . '/google/google-functions.php';
+
+$pub_date = date("c", strtotime( $pub_date ) );
+$last_build_date = date("c", strtotime( $last_build_date ) );
+?><?xml version="1.0" encoding="utf-8"?>
```

Comment:
:warning: **Warning**: Possible use of short open tags detected; found: &lt;?xml version=&quot;1.0&quot; encoding=&quot;utf-8&quot;?&gt;\n (*Generic.PHP.DisallowShortOpenTag.PossibleFound*).